lib/rot.rb

```diff
@@ -0,0 +1,26 @@
+
+class Rot
+    ALPHABET = ("a".."z").to_a + ("A".."Z").to_a
+    ALPHABET.freeze
+
+    def initialize(rot_key)
+        @rot_key=rot_key
```

Comment:
@djordje997 https://rubystyle.guide/#assignment-comparison - please fix all assignments.

```suggestion
        @rot_key = rot_key
```